pkg/local_object_storage/metabase/version.go

```diff
@@ -112,83 +115,118 @@ func migrateFrom2Version(db *DB, tx *bbolt.Tx) error {
 	return updateVersion(tx, 3)
 }
 
-func migrateFrom3Version(db *DB, tx *bbolt.Tx) error {
-	c := tx.Cursor()
-	pref := []byte{metadataPrefix}
-	if k, _ := c.Seek(pref); bytes.HasPrefix(k, pref) {
-		return fmt.Errorf("key with prefix 0x%X detected, metadata space is occupied by unexpected data or the version has not been updated to #%d", pref, currentMetaVersion)
-	}
-	if err := migrateContainersToMetaBucket(db.log, db.cfg.containers, tx); err != nil {
-		return err
+func migrateFrom3Version(l *zap.Logger, db *bbolt.DB, cs Containers) error {
+	var fromBkt, afterObj []byte
+	for {
+		if err := db.Batch(func(tx *bbolt.Tx) error {
```

Comment:
`db.Update`, please, you don't have any concurrency here.

Reply:
yeah, this was just for testing, forgot to revert